README.md

```diff
@@ -82,4 +82,4 @@ The database is automatically installed during server startup, but you need to e
 
 ### IRC Bot Setup
 
-Included in the repo is an IRC bot capable of relaying adminhelps to a specified IRC channel/server (replaces the older one by Skibiliano).  Instructions for bot setup are included in the /bot/ folder along with the bot/relay script itself.
\ No newline at end of file
+Included in the repo is an IRC bot capable of relaying adminhelps to a specified IRC channel/server (replaces the older one by Skibiliano).  Instructions for bot setup are included in the /bot/ folder along with the bot/relay script itself.
```

Comment:
This one is the same weird thing that happened when I tried to do it.
What actually got changed on this line?

Reply:
I have no clue but whatever.